core/src/edu/cornell/gdiac/main/view/CutsceneMode.java

```diff
@@ -0,0 +1,308 @@
+   */
+  private InputController inputController;
+
+  /**
+   * Menu containing the buttons
+   */
+  private ButtonsMenu buttonsMenu;
+
+
+  public CutsceneMode(GameCanvas canvas) {
+    active = false;
+    this.canvas = canvas;
+    this.textureList = new ArrayList<>();
+
+    // Initialize buttons menu
+    Batch batch = new SpriteBatch();
+    buttonsMenu = new ButtonsMenu(new ScreenViewport(), batch);
+    buttonsMenu.getViewport().update(Gdx.graphics.getWidth(), Gdx.graphics.getHeight(), true);
+  }
+
+  public void init(AssetDirectory directory) {
+    this.directory = directory;
+    textureIndex = 0;
+    this.pixelFont = directory.getEntry("display", BitmapFont.class);
+
+    // Populate textures list (each is itself a page)
+    textureList.add(directory.getEntry("cutscene0", Texture.class));
+    textureList.add(directory.getEntry("cutscene1", Texture.class));
+    textureList.add(directory.getEntry("cutscene2", Texture.class));
+
+    // Set up input controller
+    this.inputController = InputController.getInstance();
+    Gdx.input.setInputProcessor(inputController.getInputProcessor());
+
+    buttonsMenu.init();
+    buttonsMenu.show();
+
+    // Finally, activate the screen
+    active = true;
+  }
+
+  @Override
+  public void show() {
+    active = true;
+  }
+
+  @Override
+  public void render(float delta) {
+    if (!active) {
+      return;
+    }
+
+    // -------- Update ---------
+
+    // Read input from controller
+    assert inputController != null;
+    inputController.readInput(null, null);
+
+    // Handle inputs
+    if (inputController.didJustPressLeft()) {
+      movePrev();
+    }
+    if (inputController.didJustPressRight()) {
+      moveNext();
+    }
+    if (inputController.didJustPressEscape()) {
+      skip();
+    }
+
+    // Update buttons menu
+    if (buttonsMenu != null) {
+      buttonsMenu.act(delta);
+    }
+
+    // -------- Drawing --------
+
+    // Clear and begin canvas
+    Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+    canvas.begin();
+    // Draw the background
+    canvas.draw(textureList.get(textureIndex), Color.WHITE, 0f, 0f,
+      0f, 0f, Gdx.graphics.getWidth(), Gdx.graphics.getHeight());
+    canvas.end();
+
+    // Draw the buttons menu
+    if (buttonsMenu != null) {
+      buttonsMenu.draw();
+    }
+  }
+
+  @Override
+  public void resize(int width, int height) {
+
+  }
+
+  @Override
+  public void pause() {
+
+  }
+
+  @Override
+  public void resume() {
+
+  }
+
+  @Override
+  public void hide() {
+    active = false;
+  }
+
+  @Override
+  public void dispose() {
+    buttonsMenu.dispose();
+    buttonsMenu = null;
+  }
+
+  public void setScreenListener(ScreenListener listener) {
+    this.listener = listener;
+  }
+
+  // Takes care of moving to the previous texture slide
+  public void movePrev() {
+    assert !textureList.isEmpty();
+
+    textureIndex = Math.max(textureIndex - 1, 0);
+
+    // Move left
+    // CameraController.moveCameraLeftAsync(500);
```

Comment:
If you want to use animations here, all you really need to do is render the cutscene textures side by side, and then move the camera. I think this would look significantly better with animations, and all the camera controller work is already done for you, so I will strongly encourage you to implement this :)